.gitignore

```diff
@@ -89,4 +89,7 @@ fabric.properties
 .idea/watcherTasks.xml
 
 # output directory
-output/
\ No newline at end of file
+output/
+
+#api directory
+api/
```

Comment:
I don't think we should ignore this, because git knows it's a submodule.
It may be that you get a dirty, but then you should delete your changes in the subfolder.